Tasks/Kaplunou/Task2/Task2.py

```diff
@@ -0,0 +1,83 @@
+import time
+Choice=input("do you want input time or use current time? Yes or No: ")
+if Choice == "Yes":
+    Time = input("Input time in format hh:mm: ")
+    Time=Time.split(":")
+    CurentHour = int(Time[0])
+    CurentMinute = int(Time[1])
```

Comment:
you could have checked at least the length of the inputted value and a presence of ':' symbol
validation is very important, -1 point for the unwise choice